src/playbooks/with_build/upgrade_vcins_inplace.yml

```diff
@@ -0,0 +1,34 @@
+- hosts: vcin_inplace_upgrade_nodes[0]
+  gather_facts: no
+  any_errors_fatal: true
+  run_once: true
+  pre_tasks:
+    - name: Fail if vsd_migration_iso_path is not defined
+      assert:
+       that: vsd_migration_iso_path != "MISSING"
```

Comment:
Is the vsd_migration_iso applicable to vcin?

Reply:
Yes it is applicable